opm/material/fluidmatrixinteractions/EclMultiplexerMaterial.hpp

```diff
@@ -544,29 +544,30 @@ class EclMultiplexerMaterial : public TraitsT
      * error. (But not calling it will still work.)
      */
     template <class FluidState>
-    static void updateHysteresis(Params& params, const FluidState& fluidState)
+    static bool updateHysteresis(Params& params, const FluidState& fluidState)
```

Comment:
For what it's worth, this will fall off the end of the function, and return some random value, in the (unlikely, probably impossible) case that params.approach() does not match any of the listed cases. We can either add a final
```C++
return false;
```
just before the closing } of the function body or maybe throw an exception in that case?